layouts/joomla/form/field/subform/repeatable-table.php

```diff
@@ -60,8 +60,10 @@
 
     $sublayout = 'section-byfieldsets';
 } else {
-    foreach ($tmpl->getGroup('') as $field) {
-        $table_head .= '<th scope="col" style="width:45%">' . strip_tags($field->label);
+    $fields = $tmpl->getGroup('');
+
+    foreach ($fields as $field) {
+        $table_head .= '<th scope="col" style="width:' . (92/count($fields)) .'%">' . strip_tags($field->label);
```

Comment:
```suggestion
        $table_head .= '<th scope="col" style="width:' . (92 / count($fields)) . '%">' . strip_tags($field->label);
```
Seems I missed some code style errors with my previous review. But this should do it now.